GliaWidgets/Glia.swift

```diff
@@ -1,35 +1,147 @@
 import UIKit
 import SalemoveSDK
 
+/// Kind of an engagement.
 public enum EngagementKind {
+    /// No engagement
     case none
+    /// Chat
     case chat
+    /// Audio call
     case audioCall
+    /// Video call
     case videoCall
 }
 
+/// An event providing engagement state information.
 public enum GliaEvent {
+    /// Session was started
     case started
+    /// Engagement kind changed
     case engagementChanged(EngagementKind)
+    /// Session has ended
     case ended
+    /// Engagement window was minimized
     case minimized
+    /// Engagement window was maximized
     case maximized
 }
 
+/// Used to provide `UIWindowScene` to the framework.
 public protocol SceneProvider: class {
     @available(iOS 13.0, *)
     func windowScene() -> UIWindowScene?
 }
 
+/// Glia's engagement interface.
```

Comment:
You can leave it here for now, and I will add the integration guides to our public docs myself.